fury/ui/core.py

```diff
@@ -785,6 +785,9 @@ def _add_to_scene(self, scene):
             self.background.resize(size)
         scene.add(self.background, self.actor)
 
+        self.justification = self.justification
+        self.vertical_justification = self.vertical_justification
```

Comment:
I am calculating the relative position of the text using the background `Rectangle2D` which is set once the UI is added to the scene. That is why I need to perform this justification after the element is added to the scene.

Reply:
it is strange to me. I can test it only on friday. What happens when you move the RectangleUI or when you resize it?

Does the justification stay in place ?

Reply:
Yes the justification stay in place when we resize it!